libpyclingo/pyclingo.cc

```diff
@@ -6320,7 +6320,7 @@ active; you must not call any member function during search.)";
         CHECK_BLOCKED("solve");
         Py_XDECREF(stats);
         stats = nullptr;
-        static char const *kwlist[] = {"assumptions", "on_model", "on_statistics", "on_finish", "yield_", "async", nullptr};
+        static char const *kwlist[] = {"assumptions", "on_model", "on_statistics", "on_finish", "yield_", "async_", nullptr};
```

Comment:
```suggestion
        static char const *kwlist[] = {"assumptions", "on_model", "on_statistics", "on_finish", "yield_", "async", "async_", nullptr};
```
One could also accept both `async` and `async_` for backward compatibility. But then I also broke the API before...

Reply:
I cannot promise a new release by January 2019 but will keep this in mind. Right now there are only the things in the issue tracker I still want to add because some of our systems need them.